### PR TITLE
UI/statement of faith

### PR DESCRIPTION
**Pull request by Wetooa**
- added numbers in the statement of faith
- used another font

--problema--
* the font strokes don't perfectly match those in the design. The strokes are closer to the middle, making the numbers look thinner

### CODE REVIEW

File: components/about/StatementDetail.tsx [hunk 1]
```diff
@@ -1,16 +1,22 @@
-import React from 'react'
+import React from "react";
 
 interface PropsInterface {
-    className?: string
-    info: string
+  className?: string;
+  info: string;
+  number: string;
```
Comment (ezerssss) [inline components/about/StatementDetail.tsx:6]: kaning statement detail wala nani gi use anywhere else? like what if ato ni i reuse pero wala lay number? or ari rajud ni sa tong Statement nga component?

File: pages/_app.tsx [hunk 2]
```diff
@@ -1,16 +1,23 @@
...
+  subsets: ["latin-ext"],
+  style: "normal",
+  variable: "--font-garamond",
+  weight: "500",
+});
 
 export default function App({ Component, pageProps }: AppProps) {
   return (
-    <main className={`${montserrat.variable} font-sans`}>
+    <main className={`${montserrat.variable} ${garamond.variable} font-sans`}>
```
Comment (ezerssss) [inline pages/_app.tsx:19]: imo lagi gi use in the same time ang font? walay way nga i pick lang for the numbers? kay kani ga cascade raba ang style so ma apply sa everything

Comment (Wetooa) [reply pages/_app.tsx:19]: aahh ic ic, ako gi think kay needed ibutang dira ang font para maka create kog new tailwind utility class

File: components/about/StatementDetail.tsx (hunk 1 above)
Comment (Wetooa) [reply components/about/StatementDetail.tsx:6]: i believe wala na siya gi use anywhere. unsaon para dili needed i prop drill ang number?

Comment (ezerssss) [reply components/about/StatementDetail.tsx:6]: oks raman i prop drill, pero i optional prop siya IF ever gi use siya anywhere. do u think nga in the future magamit tani nga component anywhere else?

File: pages/_app.tsx (hunk 2 above)
Comment (ezerssss) [reply pages/_app.tsx:19]: actually not sure sad, what did documentation say diay?

Comment (Wetooa) [reply pages/_app.tsx:19]: hmm what if dili nalang nako i himo ug utility class ang instead import nalang in the component itself?

Comment (Wetooa) [reply pages/_app.tsx:19]: when using tailwind kay in ana on. I prop drill ang font.variable para maka make ug utility class. pero yea we can also just not create a new utility class raman sad then i use ra siya sa component an dayun

File: components/about/StatementDetail.tsx (hunk 1 above)
Comment (Wetooa) [reply components/about/StatementDetail.tsx:6]: ahhh ic ic, i don't think ma use ni na component na elsewhere.

Comment (ezerssss) [reply components/about/StatementDetail.tsx:6]: ok resolved

File: pages/_app.tsx (hunk 2 above)
Comment (ezerssss) [reply pages/_app.tsx:19]: oks ra diay drian ka `font-garamond` man diay ka, sorry wa ko kita hahaha, resolved

**Review by ezerssss — APPROVED** [review]

**Pull request merged**